src/components/scenes/FioStakingOverviewScene.tsx

```diff
@@ -81,10 +83,10 @@ export const FioStakingOverviewSceneComponent = (props: Props) => {
   }, [stakingStatus, currencyDenomination])
 
   const handlePressStake = () => {
-    Actions.jump('fioStakingChange', { change: 'add', currencyCode, walletId })
+    navigation.jump('fioStakingChange', { change: 'add', currencyCode, walletId })
```

Comment:
No such thing as `navigation.jump`, replace with `navigation.navigate`. Do a search and replace for anywhere you missed (so far there are just two in FioStatkingOverviewScene, but this comment stands to address all of the occurances).